in_progress/mission-statement.md

```diff
@@ -0,0 +1,53 @@
+
+`Scverse` is a community-based ecosystem of tools with users and developers across many organizations.
+To sustain the utility and growth of the ecosystem, it’s essential that the core analytic tools are robust and well-maintained. Accomplishing this entails consistently supporting and improving tools beyond what can be done in the conventional single-lab academic setting.
+
+Scverse constitutes an environment of tools which is deliberately not unified under a single package, but instead is a consortium of core analytic tools with shared maintenance responsibilities.
+Tools within this environment that are considered "core" for sharing data between tools or are required for fundamental analyses of single cell datatypes are placed under shared maintainance in the `scverse` github organization.
+
+Workflows in this environment are heterogeneous and this core can be supplemented with analysis specific accessory packages.
+This document lays out our vision and processes around the project's community in a transparent and accessible way.
+We are committed to keeping this community open to third-party contributions and usage and are committed to acknowledging contributions.
+This white paper may serve as an entry point for third parties into the structure of the scverse environment.
+
+## User engagement
+
+The ecosystem centred on `scverse` core packages consists of python packages which are documented on readthedocs and commonly accompanied by scientific publications.
+Its primary purpose is to allow users to generalise workflows to their own data analysis settings.
+We promote user engagement via the following mechanisms:
+
+* Clear documentation of code and usage of tutorial notebooks.
+* Participation in development though issues and pull requests.
+* Community forums, such as our Discourse and Twitter account. scverse environment-centric workshops at which scverse workflows are taught and hackathons at which functionalities are contributed. We share event dates and resources within the environment to increase impact and to exploit synergies.
+
+## Developer engagement
+
+Scverse heavily benefits from a variety of people contributing code via different channels. The channels used at the moment are:
+
+* Pull requests on scverse core or ecosystem packages: These can be used to fix bugs that are experienced during a workflow or to contribute functionalities. 
+* Feedback/bug reporting through issues and the community forum.
+* Creating and sharing tutorial content, either through written documentation or in person workshops.
+* Contributing and maintaining documentation, making it easier for people to use or understand available tools.
+* Participation in community forums, by helping new users or participating in discussions.
+* Ecosystem packages: Development of tools and analyses which interact with core packages, work with AnnData or MuData objects, or further what can be done with single cell data in Python.
```

Comment:
```suggestion
Community contributions are instrumental to scverse. Apart from direct contributions of code through pull-requests to scverse core or ecosystem packages, we emphasize the importance of

    providing feedback and reporting bugs through the community forum and the packages' issue trackers,
    creating and sharing tutorial content, either through written documentation or in-person workshops,
    contributing and maintaining documentation, making it easier for people to use or understand available tools,
    participation in community forums, by helping new users or participating in discussions on what opportunities or challenges lie ahead for scverse, and
    developing ecosystem packages, that interact with scverse core packages and data-structures and further what can be done with single-cell data in Python.
```

Reply:
Ping @grst, I feel like there was a formatting issue here. Is this what you wanted it to look like?

Reply:
huh, the bullet points vanished :scream: 
Fixed my suggestion now.